visual-dotnet/SauceLabs.Visual/FullPageConfig.cs

```diff
@@ -0,0 +1,19 @@
+using SauceLabs.Visual.GraphQL;
+
+namespace SauceLabs.Visual
+{
+    public class FullPageConfig
```

Comment:
Do we need to have two classes:
- FullPageConfigIn 
- FullpageConfig

if in the end we want to only use FullpageConfig and we know how to serialize it correctly?

Reply:
Idea was to avoid `.GraphQL` package to have a dependency to an higher package + not exposing raw content of `FullPageConfigIn` to user (since it does not provides the `Enable` flag natively.